examples/seismic/model.py

```diff
@@ -336,7 +336,7 @@ def dm(self):
         as `dm = div(m)`.
         """
         dm = Function(name="dm", grid=self.grid, space_order=self.space_order)
-        Operator(Eq(dm, div(self.m)))()
+        Operator(Eq(dm, div(self.vp)), subs=self.spacing_map)()
```

Comment:
I guess the fix is self.m -> self.vp ?

Reply:
Yes, `m=1/vp**2` which creates `nan` at the edge( #1250 )